File: Logic_HandsOn/Java/Problems on Array,Pattern etc/Program4.java
//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program4:
//		Accept one character from user and convert case of that character.
//
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////


import java.util.Scanner;

class Solution
{
	
	void DisplayConvert(char cValue)
	{
		if(cValue>='a' && cValue<='z')
		{
			System.out.println("Output: "+(char)(cValue-32));
		}
		else if(cValue>='A' && cValue<='Z')
		{
			System.out.println("Output: "+(char)(cValue+32));
		}
	}
}

class Program4
{
	public  static void main(String[] args)
	{
		Scanner s = new Scanner(System.in);
		System.out.println("Enter character: ");
		char ch = s.next().charAt(0);

		Solution sObj = new Solution();
		sObj.DisplayConvert(ch);
	}
}

/*
OUTPUT:

Enter character:
A
Output: a

*/

File: Logic_HandsOn/C++/Problems on Digits,String,Array,Pointers etc/program4.cpp
//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program4:
//		Accept one character from user and convert case of that character.
//
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////


#include<iostream>
using namespace std;


class Solution
{
	public:
	void DisplayConvert(char cValue)
	{
		if(cValue>='a' && cValue<='z')
		{
			cout<<"Output: "<<cValue-32<<"\n";
		}
		else if(cValue>='A' && cValue<='Z')
		{
			cout<<"Output: "<<cValue+32<<"\n";
		}
	}
};	


int main(){

	char cValue='\0';

	cout<<"Enter character\n";
	cin>>cValue;

	Solution *sObj = new Solution();
	sObj->DisplayConvert(cValue);
}

/*
OUTPUT:

Enter character
a
Output: A

*/
